spyder/plugins/variableexplorer/widgets/tests/test_namespacebrowser.py

```diff
@@ -203,5 +204,51 @@ def test_filtering_with_large_rows(namespacebrowser, qtbot):
     assert model.rowCount() == 1
 
 
+def test_namespacebrowser_plot_with_mute_inline_plotting_true(
+        namespacebrowser, qtbot):
+    """
+    Test that plotting a list from the namespace browser sends a signal
+    with the plot if `mute_inline_plotting` is set to `True`.
+    """
+    CONF.set('plots', 'mute_inline_plotting', True)
```

Comment:
```suggestion
    namespacebrowser.set_conf('mute_inline_plotting', True, section='plots')
```

`NamespaceBrowser` has `set_conf`/`get_conf`.